### DISCUSSION TITLE
Kitty after upgrade to 0.36 version some fonts not working

### DISCUSSION THREAD

**Discussion by ciscohack** [kovidgoyal/kitty#7735] [General]
Hi Kovid,

I just did an update to latest kitty 0.36 version. I am running kitty on macos 14.6 and some fonts now stopped working specially my favorite font "Liga SFMono Nerd Font" which i was using in previous build and working flawless but now not working. 

Also I have lots of nerd font install and kitten choose-font command showing very few font only and enter to select the font also not working. infact when i hit enter nothing happening and my screen stuck and ctrl+c and ctrl+z or ESC button nothing work to comeout of prompt foce me to kill the terminal.

can you please how i can use my old font back in new kitty as it was working on older fonts

**Comment by kovidgoyal** [comment]
First of all, dont use patched fonts.
Then follow the instructions at: https://sw.kovidgoyal.net/kitty/faq/#kitty-is-not-able-to-use-my-favorite-font

**Reply by ciscohack** [reply]
@kovidgoyal Thanks for quick response. Yes i know you have mentioned earlier that kitty not need patched font but the font i mentioned was working fine in older kitty ..it just stopped now and i tried without patched SFmono font that is also not working..

I tried font.conf file option but that is also not working

why kitten choose font enter not selecting font with enter and stuck font selection prompt and force me to kill the terminal

**Comment by kovidgoyal** [comment]
On Sun, Aug 18, 2024 at 12:24:21AM -0700, ciscohack wrote:
> @kovidgoyal Thanks for quick response. Yes i know you have mentioned earlier that kitty not need patched font but the font i mentioned was working fine in older kitty ..it just stopped now and i tried without patched SFmono font that is also not working..

As is mentioned in the link a font will work in kitty ONLY if it shows
up under the fixed width collection in Font Book.


>
> I tried font.conf file option but that is also not working

font.conf is for LInux on macOS there is no way to override monospace
detection.

>
> why kitten choose font enter not selecting font with enter and stuck font selection prompt and force me to kill the terminal

I have no idea what that means. choose-fonts works fine for me and
doesnt get stuck anywhere. Simply follow the instructions on the screen.

**Reply by kovidgoyal** [reply]
And I just tried SF Mono from Apple and it works fine in kitty 0.36. It shows up in the choose-fonts kitten and after selecting it and following the instructions, it becomes the kitty font.

**Reply by kovidgoyal** [reply]
Oh and if for some reason you cant use the choose fonts kitten, you can always just set font_family to `SF Mono` in kitty.conf and it works, as evidenced by:

```
kitty --config=NONE -o "font_family=SF Mono" --debug-font-fallback
```

**Reply by ciscohack** [reply]
@kovidgoyal I got your point that kitty will only use fixed font but same font was working in all older release except 0.36 what change either older kitty does not have this restriction or it's now strictly checking and not using the font ..is what i am trying to complain

**Comment by kovidgoyal** [comment]
On Sun, Aug 18, 2024 at 12:43:40AM -0700, ciscohack wrote:
> @kovidgoyal I got your point that kitty will only use fixed font but same font was working in all older release except 0.36 what change either older kitty does not have this restriction or it's now strictly checking and not using the font ..is what i am trying to complain

If it worked in older kitty that was a bug. A bug that has now been
fixed.

**Reply by p7r0x7** [reply]
If this is true, that's fine; I'm having this issue as I am also using a pre-patched nerd font.

**Reply by p7r0x7** [reply]
> [0.177] The font CaskaydiaMono Nerd Font was not found, falling back to Menlo

This is what is printed now running kitty from a shell.

**Reply by p7r0x7** [reply]
Following your recommendation made it work.

**Comment by diegoulloao** [comment]
Geist Mono stopped working, also GeistMono Nerd Font.
I'm not able to get it working...

**Comment by ciscohack** [comment]
@diegoulloao try "GeistMono Nerd Font Mono" this version should work

**Reply by diegoulloao** [reply]
Actually it doesn't render variants well. I tried to add Light as `font_family` and SemiBold for `font_bold`, it doesn't vary.
Geist Mono does. I downgraded to `0.35.2`

**Reply by ciscohack** [reply]
may i have screenshot as reference .... this font is very bold in my terminal ..semibold would be too thick

**Reply by diegoulloao** [reply]
> may i have screenshot as reference .... this font is very bold in my terminal ..semibold would be too thick

even "GeistMono Nerd Font Mono" doesn't work anymore with latest version.

**Comment by diegoulloao** [comment]
Was someone able to get the font working properly?
